### ISSUE TITLE
Unrecoverable error in the server. ChannelFailures: TypeError("'NoneType' object has no attribute '__getitem__'",)

### ISSUE THREAD

**Issue by YipYup** [sabnzbd/sabnzbd#472]
v1.0.0 RC1

Unrecoverable error in the server.
Traceback (most recent call last):
  File "/usr/share/sabnzbdplus/cherrypy/_cpwsgi.py", line 180, in trap
    return func(*args, **kwargs)
  File "/usr/share/sabnzbdplus/cherrypy/_cpwsgi.py", line 99, in __call__
    return self.nextapp(environ, start_response)
  File "/usr/share/sabnzbdplus/cherrypy/_cpwsgi.py", line 410, in tail
    return self.response_class(environ, start_response, self.cpapp)
  File "/usr/share/sabnzbdplus/cherrypy/_cpwsgi.py", line 234, in __init__
    self.run()
  File "/usr/share/sabnzbdplus/cherrypy/_cpwsgi.py", line 306, in run
    request, resp = self.cpapp.get_serving(local, remote, scheme, sproto)
  File "/usr/share/sabnzbdplus/cherrypy/_cptree.py", line 132, in get_serving
    cherrypy.engine.publish('acquire_thread')
  File "/usr/share/sabnzbdplus/cherrypy/process/wspbus.py", line 223, in publish
    raise exc
ChannelFailures: TypeError("'NoneType' object has no attribute '__getitem__'",)

**Comment by Safihre** [comment]
What OS? On startup this is, right?

**Comment by YipYup** [comment]
Ubuntu 14.04 LTS 64-bit.
Yes, I believe it is during startup. Issue went away within a minute or so, I suspect once it finished loading.

**Comment by thezoggy** [comment]
sounds like: http://forums.sabnzbd.org/viewtopic.php?f=2&t=4404&p=31754

you have your own instance of cherrypy installed?

**Comment by thezoggy** [comment]
@shypike  sab doesnt have the ability to turn on cherrypy logging? (should add that feature from sickbeard)

https://github.com/midgetspy/Sick-Beard/blob/development/sickbeard/config.py#L114

**Comment by YipYup** [comment]
Would I add this to "EXTRAOPTS=" within /etc/default/sabnzbdplus , or to "DEFOPTS=" within /etc/init.d/sabzbdplus ?

**Comment by shypike** [comment]
EXTRAOPTS

**Comment by shypike** [comment]
@Safihre Can this be caused by the new pre-loading feature?
Although it looks like a bug in the start-up code of CherryPy.

**Comment by Safihre** [comment]
I would highly doubt it was the preloading, since that's only called and executed after a page is requested and the template is rendered. 
So not on load of the server, does it happen when you open or re-load the webpage @YipYup?

**Comment by YipYup** [comment]
@Safihre The application had previously been running. I had the browser session open, then accessed the OS via SSH to reload the application. This error happened when re-using the browser session against the application immediately after I had restarted the service. The only thing I can't remember is exactly what I was looking at in the web app / which section of the web interface that I had accessed to generate it.
